### ISSUE TITLE
can't login in new installation, Debian 10 through apt repository

### ISSUE THREAD

**Issue by gentooza** [phpmyadmin/phpmyadmin#17706]
### Describe the bug

When I try to login with user X I get a blank screen.
In apache logs I get:
```
AH01071: Got error 'PHP message: PHP Fatal error:  Uncaught mysqli_sql_exception: SELECT command denied to user 'X'@'localhost' for table 'user' in /usr/share/phpmyadmin/libraries/classes/Dbi/DbiMysqli.php:199
Stack trace:
#0 /usr/share/phpmyadmin/libraries/classes/Dbi/DbiMysqli.php(199): mysqli->query()
#1 /usr/share/phpmyadmin/libraries/classes/DatabaseInterface.php(316): PhpMyAdmin\\Dbi\\DbiMysqli->realQuery()
#2 /usr/share/phpmyadmin/libraries/classes/DatabaseInterface.php(2390): PhpMyAdmin\\DatabaseInterface->tryQuery()
#3 /usr/share/phpmyadmin/libraries/classes/DatabaseInterface.php(2332): PhpMyAdmin\\DatabaseInterface->isUserType()
#4 /usr/share/phpmyadmin/libraries/server_common.inc.php(43): PhpMyAdmin\\DatabaseInterface->isSuperuser()
#5 /usr/share/phpmyadmin/index.php(116): include('...')
#6 {main}
thrown in /usr/share/phpmyadmin/libraries/classes/Dbi/DbiMysqli.php on line 199
PHP message: PHP Fatal error:  Uncaught mysqli_sql_exception: SELECT command denied to user 'X'@'localhost' for table 'user' in /usr/share/phpmyadmin/libraries/classes/Dbi/DbiMysqli.php:199
Stack trace:
#0 /usr/
```

user exists, it's webpage works ok, and I can login using mariadb console

### To Reproduce

install Debian 10 (php7.3, etc.)
install phpmyadmin (5.0.4) using the backports repository
I found problems for using the socket to connect to mariadb, as it's path has changed, I don't remember now what I changed
try to use it

### Expected behavior

User can login as expected.

If wrong password supplied or something else, a screen showing up the error, not a blank page"

### Screenshots

If applicable, add screenshots to help explain the bug.

### Server configuration

- Operating system:Debian 10 GNU/Linux
- Web server: Apache 2.4.38
- Database version: Mariadb 10.3.34
- PHP version: 7.3.33
- phpMyAdmin version: 5.0.4

### Client configuration

- Browser: Firefox
- Operating system: GNU/Linux

### Additional context
Thanks!

**Comment by williamdes** [comment]
Hi @gentooza
Could you use buster backports to upgrade your phpmyadmin?
Ref: [Wiki for install methods on Ubuntu/Debian](https://github.com/phpmyadmin/phpmyadmin/wiki/DebianUbuntu#related-issues-and-comments-on-install-methods)

That said I am not sure it will fix the errror, we will probably have to investigate

**Comment by williamdes** [comment]
Indeed, it's up to date `4:5.0.4+dfsg2-2~bpo10+1` (bpo means backports)
We will investigate your report then

**Comment by MauricioFauth** [comment]
This is weird, the `mysqli_sql_exception` shouldn't be thrown.

Could you change the file `/usr/share/phpmyadmin/libraries/classes/Dbi/DbiMysqli.php` and add `mysqli_report(MYSQLI_REPORT_OFF);` at the line 62 to avoid the `mysqli_sql_exception` to be thrown?

```diff
         }
+        mysqli_report(MYSQLI_REPORT_OFF);
         $mysqli = mysqli_init();
```

https://github.com/phpmyadmin/phpmyadmin/blob/582023bf8e24bcf93a859ea7faa07a8863fad8d3/libraries/classes/Dbi/DbiMysqli.php#L61-L63

Related to eb2d86574043e4c1c468888115d0e51245ac6ee7.

**Comment by gentooza** [comment]
Hi @MauricioFauth !

yes! this did the trick.
Now I have lots of deprecation notices, do I share them?
I do whatever you need to debug, of course.

cheers

**Comment by williamdes** [comment]
You could update to 5.1.4 if you install the Ununtu PPA. It will solve the deprecations. That said I am working on sending 5.1 into buster backports but some elements are blocking me so it will take some time.
Ref: [Wiki for install methods on Ubuntu/Debian](https://github.com/phpmyadmin/phpmyadmin/wiki/DebianUbuntu#related-issues-and-comments-on-install-methods)

**Comment by MauricioFauth** [comment]
> yes! this did the trick. Now I have lots of deprecation notices, do I share them? I do whatever you need to debug, of course.

That's strange. Could you verify that phpMyAdmin is really using PHP 7.3? The default mysqli error mode was only changed in PHP 8.1. That will explain the deprecation notices, since phpMyAdmin 5.0 does not fully support PHP 8.1, as it didn't exist back then.

**Comment by gentooza** [comment]
This was the problem!

I'm really sorry, I though 7.3 was my default php version as I set it using update-alternatives:
```
update-alternatives --config php
update-alternatives --config php-cgi
```

But inserting a:  echo 'PHP version: ' . phpversion(); in phpmyadmin header shown me 8.1!

So I had system not clean enough, I've removed from apache 8.1 and 8.0 fpm configurations
```
a2disconf php8.1-fpm
a2disconf php8.0-fpm
```
letting only 7.3 and now is working perfect.

I'm really sorry again for the waste of time
thanks a lot (closing)
